### DISCUSSION TITLE
请问docker版如何配置运行，pan93412/unblock-netease-music-enhanced 里面没找到说明

### DISCUSSION THREAD

**Discussion by michealhansun** [1715173329/UnblockNeteaseMusic#268] [Q&A] [answered]
请问docker版如何配置运行，pan93412/unblock-netease-music-enhanced 里面没找到说明

**Comment by pan93412** [comment, marked as answer]
https://github.com/UnblockNeteaseMusic/server#docker-hub

已經在 README.md 加入完使用指引了～假如這有解決到你的問題，幫我點一下 `Mark as answer`。謝謝！
